openapi/v2/openapi.json

```diff
@@ -464,44 +502,90 @@
+        },
+        "message": {
+          "type": "string",
+          "description": "Contains a text giving more details of the condition. This will usually be progress reports, or error messages, and\nare intended for use by humans, to debug problems."
+        }
+      },
+      "description": "Contains the details of a condition that describes the status of a cluster order."
+    },
+    "v1ClusterOrderConditionType": {
+      "type": "string",
+      "enum": [
+        "CLUSTER_ORDER_CONDITION_TYPE_UNSPECIFIED",
+        "CLUSTER_ORDER_CONDITION_TYPE_ACCEPTED",
+        "CLUSTER_ORDER_CONDITION_TYPE_REJECTED",
+        "CLUSTER_ORDER_CONDITION_TYPE_CANCELED",
+        "CLUSTER_ORDER_CONDITION_TYPE_FULFILLED",
+        "CLUSTER_ORDER_CONDITION_TYPE_FAILED"
+      ],
+      "default": "CLUSTER_ORDER_CONDITION_TYPE_UNSPECIFIED",
+      "description": "Types of conditions used to describe a cluster order.\n\n - CLUSTER_ORDER_CONDITION_TYPE_UNSPECIFIED: Unspecified indicates that the condition unknown.\n\nThis will never be appear in the `spec.conditions` field of a order.\n - CLUSTER_ORDER_CONDITION_TYPE_ACCEPTED: Accepted indicates that the order has been accepted by the system.\n - CLUSTER_ORDER_CONDITION_TYPE_REJECTED: Rejected indicates that the order has been rejected by the system, so no further processing will be fulfill it.\n - CLUSTER_ORDER_CONDITION_TYPE_CANCELED: Canceled indicates that the order has been canceled by the user.\n - CLUSTER_ORDER_CONDITION_TYPE_FULFILLED: Fulfilled indicates that the order has been successfully fulfilled.\n\nThe details of the resulting cluster will be available in the `cluster` object indicated by the `status.cluster_id`\nfield.\n - CLUSTER_ORDER_CONDITION_TYPE_FAILED: Failed indicates that fulfillment of the order failed.\n\nCurrently there are no sepcific `reason` values defined."
     },
     "v1ClusterOrderSpec": {
       "type": "object",
       "properties": {
         "template_id": {
           "type": "string",
-          "description": "Reference to the cluster template. This is mandatory, and must be the value of the `id` field of one of the cluster\ntemplates."
+          "description": "Reference to the cluster template.\n\nThis is mandatory, and must be the value of the `id` field of one of the cluster templates."
         }
       },
-      "description": "The spec contains the details that the user provides to request the provisioning of the cluster."
+      "description": "Contains the details that the user provides to request the provisioning of the cluster."
     },
     "v1ClusterOrderState": {
       "type": "string",
       "enum": [
         "CLUSTER_ORDER_STATE_UNSPECIFIED",
-        "CLUSTER_ORDER_STATE_ACCEPTED",
-        "CLUSTER_ORDER_STATE_REJECTED",
-        "CLUSTER_ORDER_STATE_CANCELED",
+        "CLUSTER_ORDER_STATE_PENDING",
```

Comment:
I think in kubernetes this would typically be "Progressing".

Reply:
OK, I will rename that.

Reply:
Done.